_posts/2024-04-16-mai-win-sr2024.md

```diff
@@ -0,0 +1,176 @@
+
+{% assign final_full = "/images/content/blog/sr2024/final-wgs-tlc-hso-robots.large.jpg" | prepend: site.baseurl %}
+{% include figure.html
+           class="left"
+           src="/images/content/blog/sr2024/final-wgs-tlc-hso-robots.thumbnail.jpg"
+           url=final_full
+           caption="Grabbing an Asteroid in the SR2024 Final"
+           alt="
+            The SR2024 Final, viewed across the arena from the corner between Planets (scoring zones) 2 & 3.
+            The venue lighting strongly illuminates the arena, which is surrounded on two sides by competitors and their supervisors.
+            A volunteer commentator in a blue shirt, wearing a multicoloured LED halo, stands between the crowd and the arena.
+            The edges of the arena are constructed from wooden panels, while the floor is blue carpet.
+            The boundaries to the Planets are marked with brightly coloured tape.
+            In the center of the frame a pink robot with a green flag for Planet 0 and large wheels holds up an Asteroid (cube shaped token).
+            Just visible behind is the Egg (another token) on its Nest (a low plinth).
+            To the left a robot with a yellow flag for Planet 3 and a tall arm heads away across the arena.
+            To the right a pink robot with a pink flag for Planet 2 heads away across the arena.
+            Scattered throughout the arena are other Asteroids -- some still on their starting positions and others moved by robots, either by accident or moved onto Planets.
+            In the background, a Spaceship (a plastic box large enough for several Asteroids) is on Planet 1.
+           "
+%}
+
+The thrilling final featured four teams which had shown consistent performance throughout the earlier stages and included teams from three schools which have previously won Student Robotics, so it promised to be exciting even before it started.
+
+In a surprise move at the start of the match Wisbech Grammar School immediately captured The Ladies’ College's Spaceship. This appeared to confuse The Ladies’ College's robot, though it soon switched to placing Asteroids on its Planet instead. Meanwhile Gymnasium Markt Indersdorf managed to get a token into their own Spaceship, giving them a commanding lead.
+
+The Ladies’ College and Headington School each captured two Asteroids, however Headington School's robot gifted The Ladies’ College a third in the late stages of the match. Wisbech Grammar School's consistent performance let them capture four Asteroids, and might have been enough to win had Gymnasium Markt Indersdorf not also captured some Asteroids into its robot.
+
+Wisbech Grammar School were the source of another tense moment near the very end of the match when they attempted to steal The Ladies’ College's first Asteroid, however they were unable to navigate off the Planet in time meaning that the Asteroid still scored for The Ladies’ College.
+
+## Results
+
+- Winner: MAI – Gymnasium Markt Indersdorf
```

Comment:
Suggestion: As this is the winners table, I think we should explicitly link out for every team, not just some of them.

Reply:
ec4778f